lib/sigstore/rekor/client.rb

```diff
@@ -103,10 +118,38 @@ def post(expected_entry)
           results = JSON.parse(resp.body)
 
           results.map do |result|
-            Transparency::LogEntry.from_response(result)
+            Entries.decode_transparency_log_entry(result)
           end.min_by(&:integrated_time)
         end
       end
+
+      def self.decode_transparency_log_entry(response)
```

Comment:
it would be nice if translation between the response we get from rekor and the tlog entries in a bundle weren't necessary... @haydentherapper do you know if there's any way around this?

Reply:
Not currently, this response structure was before we had the protobuf specs. For Rekor v2, we'll definitely use the spec compliant structure, but for now, we just provide utilities to map between the two.

Maybe we could do this in v1 via a content type header? Can you file an issue if so?